inventory-consumer/handler.go

```diff
@@ -37,15 +37,27 @@ func (this *handler) onMessage(ctx context.Context, msg kafka.Message) {
 				log.Println("Error retrieving state for account: ", value.Host.Account)
```

Comment:
Do you need to return here?  If err is not nil, will accState be a nil?  Is there any reason to continue processing this host if you cannot retrieve the account state?

Reply:
Yep that's a good catch. Will update